Cargo.toml

```diff
@@ -18,15 +18,17 @@ categories = ["asynchronous", "cryptography", "network-programming"]
 native-tls = "0.2.3"
 thiserror = "1.0.9"
 async-std = { version = "1.4.0", default-features = false, features = ["std"], optional = true }
+futures-util = { version = "0.3.1", features = ["io"], optional = true }
 tokio = { version = "0.2.9", default-features = false, features = ["io-util"], optional = true }
 
 [features]
-default = ["runtime-async-std"]
+default = ["runtime-futures"]
```

Comment:
can you revert the default please, I would like to ship this as a non breaking change